src/components/InstallModalContent.js

```diff
@@ -1,36 +1,102 @@
+    textField.innerText = this.command();
+    document.body.appendChild(textField);
+    textField.select();
+    document.execCommand("copy");
+    textField.remove();
+    e.target.focus();
+    // sechedules a call to setState to hide the popover in 5 seconds
+    setTimeout(
+      function() {
+        this.setState({ copied: false });
+      }.bind(this),
+      5000
+    );
+    this.setState({ copied: true });
+  };
+
+  command() {
+    return `kubectl create -f ${this.props.expcrdurl}`;
   }
 
   render() {
     return (
       <div className="install-modal-content">
         <div className="modal-header">
-          <div className="modal-img-placeholder"> <img src={this.props.logo} /></div>
+          <div className="modal-img-placeholder">
+            {" "}
+            <img src={this.props.logo} />
+          </div>
           <div className="modal-header-title-container">
             <h1 className="modal-title">{this.props.displayName}</h1>
-            <span className="modal-subtitle">Provided by {this.props.provider}</span>
+            <span className="modal-subtitle">
+              Provided by {this.props.provider}
+            </span>
           </div>
         </div>
 
         <h3 className="install-title">Install the Chaos Experiments</h3>
 
-        <div className="modal-install-instruction"><span className="modal-install-instruction-number"></span>You can install the Chaos Experiments by following command</div>
-        <div className="modal-code">kubectl create -f  {this.props.expcrdurl}</div>
-        <div className="modal-install-instruction"><span className="modal-install-instruction-number gray-text-color">Notes:</span></div>
+        <div className="modal-install-instruction">
+          <span className="modal-install-instruction-number"></span>You can
+          install the Chaos Experiments by following command
+        </div>
+        <div className="modal-install-code-row">
+          <div className="modal-code">{this.command()}</div>
+          {document.queryCommandSupported("copy") && (
+            <Popover
+              // Allows the popover to show over the modal
+              containerStyle={{ zIndex: 1000 }}
+              isOpen={this.state.copied}
+              position={"bottom"}
+              content={({ position, targetRect, popoverRect }) => (
+                <ArrowContainer
+                  position={position}
+                  targetRect={targetRect}
+                  popoverRect={popoverRect}
+                  arrowColor="#b0b0b0"
+                  arrowSize={3}
+                  arrowStyle={{ opacity: 0.7 }}
+                >
+                  <div className="modal-install-popover-content">Copied</div>
+                </ArrowContainer>
+              )}
+            >
+              <button
+                onClick={this.copyToClipboard}
+                className="modal-install-clipboard"
+              >
+                <FaCopy />
```

Comment:
I'm still not sure what the icon is that you were asking for so I am using the FontAwesome copy icon here unless you can tell me where to find the icon that you're suggesting